spec/calendar.html

```diff
@@ -237,9 +237,15 @@ <h1>CalendarInLeapYear ( _calendar_, _dateLike_ )</h1>
       <h1>ToTemporalCalendar ( _temporalCalendarLike_ )</h1>
       <emu-alg>
         1. If Type(_temporalCalendarLike_) is Object, then
-          1. Return _temporalCalendarLike_.
+          1. If _temporalCalendarLike_ has an [[InitializedTemporalDate]], [[InitializedTemporalDateTime]], [[InitializedTemporalMonthDay]], [[InitializedTemporalTime]], [[InitializedTemporalYearMonth]], or [[InitializedTemporalZonedDateTime]] internal slot, then
+            1. Return _temporalCalendarLike_.[[Calendar]].
+          1. If ? HasProperty(_item_, *"calendar"*) is *false*, return _item_.
+          1. Set _item_ to ? Get(_item_, *"calendar"*).
+          1. If Type(_item_) is Object and ? HasProperty(_item_, *"calendar"*) is *false*, return _item_.
```

Comment:
can you remind me why this is checking `temporalCalendarLike.calendar.calendar`?

Reply:
The consensus is documented in https://github.com/tc39/proposal-temporal/issues/925#issuecomment-709544520 — tl;dr this is to allow `{ calendar: 'gregory', year: 2021, month: 4, day: 13 }` as an input, but not `{ calendar: { calendar: { calendar: { ... } } } }`.